crates/router/src/types/api.rs

```diff
@@ -26,7 +26,7 @@ use crate::{
     configs::settings::Connectors,
     connector, consts,
     core::errors::{self, CustomResult},
-    services::{request, ConnectorIntegration, ConnectorRedirectResponse},
+    services::{request, ConnectorIntegration, ConnectorRedirectResponse, ConnectorValidation},
```

Comment:
Yes, since we are adding this trait to Payment, this import was made to add ConnectorValidation trait for api crate itself otherwise for all file where payment is implemented, we need to import services crate too for ConnectorValidation